src/Management/test/Endpoint.Test/ActuatorRouteBuilderExtensionsTest.cs

```diff
@@ -46,10 +46,22 @@ static bool Query(Type t)
                 return t.GetInterfaces().Any(type => type.FullName == "Steeltoe.Management.IEndpoint");
             }
 
-            static bool SupportedOnPlatform(Type t)
+            static bool SupportedOnPlatform(Type type)
             {
-                return !(t.Name.StartsWith("ThreadDump") || t.Name.StartsWith("HeapDump")) || (t.Name.StartsWith("ThreadDump") && Platform.IsWindows) ||
-                    t.Name.StartsWith("HeapDump");
+                bool isThreadDump = type.Name.StartsWith("ThreadDump", StringComparison.Ordinal);
+                bool isHeapDump = type.Name.StartsWith("HeapDump", StringComparison.Ordinal);
+
+                if (!(isThreadDump || isHeapDump))
+                {
+                    return true;
+                }
+
+                if (isThreadDump && Platform.IsWindows)
+                {
+                    return true;
+                }
+
+                return isHeapDump;
             }
 
             List<Type> types = Assembly.Load("Steeltoe.Management.Endpoint").GetTypes().Where(Query).Where(SupportedOnPlatform).ToList();
```

Comment:
```suggestion
            List<Type> types = Assembly.Load("Steeltoe.Management.Endpoint").GetTypes().Where(Query).ToList();
```

Reply:
Thanks, I've already pushed a commit to address this (and keep the formatter happy).